applications/colonoscopy_segmentation/CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ if(BUILD_TESTING)
 
   # Add test
   add_test(NAME colonoscopy_segmentation_python_test
-           COMMAND python3 ${CMAKE_CURRENT_BINARY_DIR}/colonoscopy_segmentation_test.py
+           COMMAND ${PYTHON_EXECUTABLE_TEST} ${CMAKE_CURRENT_BINARY_DIR}/colonoscopy_segmentation_test.py
```

Comment:
Will we need to do this replacement for all Python-based tests? Not a blocker, can be a followup